src/storage/heap_file.c

```diff
@@ -14530,14 +14530,16 @@ heap_dump_heap_file (THREAD_ENTRY * thread_p, FILE * fp, bool dump_records, cons
   status = xlocator_find_class_oid (thread_p, class_name, &class_oid, S_LOCK);
   if (status != LC_CLASSNAME_EXIST)
     {
-      return;
+      return ER_LC_UNKNOWN_CLASSNAME;
```

Comment:
NIT:

How about using a goto statement to consistently return the error_code?

Reply:
Done.